tests/build.gradle

```diff
@@ -49,11 +48,7 @@ dependencies {
     testCompile 'io.spray:spray-json_2.11:1.3.2'
     testCompile 'com.google.code.tempus-fugit:tempus-fugit:1.2-SNAPSHOT'
     testCompile 'com.google.code.gson:gson:2.3.1'
-
-    testCompile files('$whiskhome/common/scala/build/libs/scala.jar',
-                      '$whiskhome/core/controller/build/libs/controller.jar', 
-                      '$whiskhome/core/dispatcher/build/libs/dispatcher.jar')
-    testCompile files(test_common)
+    testCompile project(':WhiskTest').sourceSets.test.output
```

Comment:
@markusthoemmes I verified it works without openwhisk and openwhisk tests compiled. 

I deleted the `build` folder under openwhisk/common/scala, openwhisk/core/controller/build, openwhisk/core/dispatcher/build and openwhisk/tests/build. And then I ran `./gradlew :tests:test` under openwhisk-catalog, it will generate all these build files and pass the compile and test.

I know it works fine without the dependencies to projects `:common:scala`, `:core:dispatcher` and `:core:controller` at present. I wonder if we will add them in the future when needed or we add them now at the beginning.

Daisy